gulpfile.babel.js

```diff
@@ -24,15 +24,16 @@
 // You can read more about the new JavaScript features here:
 // https://babeljs.io/docs/learn-es2015/
 
-import path from 'path';
 import gulp from 'gulp';
 import del from 'del';
 import runSequence from 'run-sequence';
 import browserSync from 'browser-sync';
-import swPrecache from 'sw-precache';
+import workboxBuild from 'workbox-build';
 import gulpLoadPlugins from 'gulp-load-plugins';
+import replace from 'replace';
```

Comment:
This is missing from `package.json`, and doesn't appear to be used anyway. Please remove.